set.mm

```diff
@@ -237805,6 +238079,472 @@ factorization into prime power factors (even if the exponents are
+    simpgnsgeqd.2 $e |- .0. = ( 0g ` G ) $.
+    simpgnsgeqd.3 $e |- ( ph -> G e. SimpGrp ) $.
+    simpgnsgeqd.4 $e |- ( ph -> A e. ( NrmSGrp ` G ) ) $.
+    $( A normal subgroup of a simple group is either the whole group or the
+       trivial subgroup.  (Contributed by Rohan Ridenour, 3-Aug-2023.) $)
+    simpgnsgeqd $p |- ( ph -> ( A = { .0. } \/ A = B ) ) $=
+      ( csn cpr wcel wceq wo cnsg cfv simpgnsgd eleqtrd elpri syl ) ABEJZCKZLBU
+      AMBCMNABDOPUBIACDEFGHQRBUACST $.
+  $}
+
+  ${
+    $d ph x $.  $d x .0. $.  $d x B $.  $d x G $.
+    2nsgsimpgd.1 $e |- B = ( Base ` G ) $.
+    2nsgsimpgd.2 $e |- .0. = ( 0g ` G ) $.
+    2nsgsimpgd.3 $e |- ( ph -> G e. Grp ) $.
+    2nsgsimpgd.4 $e |- ( ph -> -. { .0. } = B ) $.
+    2nsgsimpgd.5 $e |- ( ( ph /\ x e. ( NrmSGrp ` G ) ) ->
+      ( x = { .0. } \/ x = B ) ) $.
+    $( If any normal subgroup of a nontrivial group is either the trivial
+       subgroup or the whole group, the group is simple.  (Contributed by Rohan
+       Ridenour, 3-Aug-2023.) $)
+    2nsgsimpgd $p |- ( ph -> G e. SimpGrp ) $=
+      ( wcel wa wceq adantl simpr syl adantr eqeltrd adantlr cvv cfv csn cpr cv
+      cnsg c2o cen wo elprg mpbird cgrp 0nsg nsgid elpri mpjaodan impbida eqrdv
+      wb snex a1i cbs fvexi enpr2d eqbrtrd issimpgd ) ADHADUEUAZEUBZCUCZUFUGABV
+      FVHABUDZVFKZVIVHKZAVJLVKVIVGMZVICMZUHZJVJVKVNURAVIVGCVFUINUJAVKLVLVJVMAVL
+      VJVKAVLLVIVGVFAVLOAVGVFKZVLADUKKZVOHDEGULPQRSAVMVJVKAVMLVICVFAVMOACVFKZVM
+      AVPVQHCDFUMPQRSVKVNAVIVGCUNNUOUPUQAVGCTTVGTKAEUSUTCTKACDVAFVBUTIVCVDVE $.
+  $}
+
+  ${
+    $d ph x $.  $d x .0. $.  $d x B $.  $d x G $.
+    simpgnsgbid.1 $e |- B = ( Base ` G ) $.
+    simpgnsgbid.2 $e |- .0. = ( 0g ` G ) $.
+    simpgnsgbid.3 $e |- ( ph -> G e. Grp ) $.
+    simpgnsgbid.4 $e |- ( ph -> -. { .0. } = B ) $.
+    $( A nontrivial group is simple if and only if its normal subgroups are
+       exactly the group itself and the trivial subgroup.  (Contributed by
+       Rohan Ridenour, 4-Aug-2023.) $)
+    simpgnsgbid $p |- ( ph ->
+    ( G e. SimpGrp <-> ( NrmSGrp ` G ) = { { .0. } , B } ) ) $=
+      ( vx csimpg wcel cnsg cfv csn cpr wceq wa simpr simpgnsgd adantr wn cv wo
+      cgrp simplr eleqtrd elpri syl 2nsgsimpgd impbida ) ACJKZCLMZDNZBOZPZAUKQB
+      CDEFAUKRSAUOQZIBCDEFACUDKUOGTAUMBPUAUOHTUPIUBZULKZQZUQUNKUQUMPUQBPUCUSUQU
+      LUNUPURRAUOURUEUFUQUMBUGUHUIUJ $.
+  $}
+
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+  Classification of abelian simple groups
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+$)
+
+  ${
+    $d ph n $.  $d A n $.
+    cycsubggend.1 $e |- B = ( Base ` G ) $.
+    cycsubggend.2 $e |- .x. = ( .g ` G ) $.
+    cycsubggend.3 $e |- F = ( n e. ZZ |-> ( n .x. A ) ) $.
+    cycsubggend.4 $e |- ( ph -> A e. B ) $.
+    $( The cyclic subgroup generated by ` A ` includes its generator.
+       (Contributed by Rohan Ridenour, 3-Aug-2023.) $)
```

Comment:
"Although this theorem holds for any class ` G ` , the definition of ` F ` is only meaningful if ` G ` is a group." should be added to the comment (see also ~ cycsubmcl).

Reply:
Added that remark.